tests/TestCase/Controller/Component/SecurityComponentTest.php

```diff
@@ -488,7 +488,6 @@ public function testValidatePostIgnoresCsrfToken(): void
         $debug = 'not used';
 
         $this->Controller->setRequest($this->Controller->getRequest()->withParsedBody([
-            '_csrfToken' => 'abc123',
```

Comment:
The `_csrfToken` input would still be in request data.

Reply:
Clearing `_csrfToken` from request data is `CsrfProtectionMiddleware`'s job.

https://github.com/cakephp/cakephp/blob/6fec49c1462802c700aaf7fdd0dbf3a0ff92a36e/src/Http/Middleware/CsrfProtectionMiddleware.php#L162-L166